### DISCUSSION TITLE
Default Polygon provider produces warnings on MetaMask when prompting user to create new network

### DISCUSSION THREAD

**Discussion by jcislo** [rainbow-me/rainbowkit#1459] [General]
Please see image. Metamask doesn't seem to like that the default rainbow kit provider for Polygon sets network name to "Polygon" and not "Polygon Mainnet", throwing a warning when prompting the user to create the new network.

Everything here still works and is functional, but I am getting this complaint from a client regarding UX and how it could throw off users new to blockchain in general (those setting up a wallet for their first time).

This can be (should be) fixed by using a custom provider in your _app.tsx to set the provider name as "Polygon Mainnet", however it would be nice if the default settings from rainbowkit were fully compatible with metamask.

![image](https://github.com/rainbow-me/rainbowkit/assets/55262056/4e2cedde-f079-4de8-8ed8-8f01a48cc9b9)

**Comment by DanielSinclair** [comment]
Thanks for spotting. That's an unfortunate decision from MetaMask that really doesn't offer any security value, but we can investigate and get this fixed. Opened #1489 to track the issue.

**Reply by robwilkes** [reply]
The same thing happens with zkSync Era..
It's called zkSync Era Mainnet in MetaMask, and hence raises warning in MetaMask.
